examples/data-block-examples/landsat8-geotiff-example.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
Please add an introductory text pointing out what the used blocks are and do. You can check other notebooks in the same directory for examples.

Reply:
Just added the description, however I will update it the new notebook.